vendor/github.com/riobard/go-virtualbox/machine.go

```diff
@@ -23,21 +23,25 @@ type Flag int
 
 // Flag names in lowercases to be consistent with VBoxManage options.
 const (
-	F_acpi Flag = 1 << iota
-	F_ioapic
-	F_rtcuseutc
-	F_cpuhotplug
-	F_pae
-	F_longmode
-	F_synthcpu
-	F_hpet
-	F_hwvirtex
-	F_triplefaultreset
-	F_nestedpaging
-	F_largepages
-	F_vtxvpid
-	F_vtxux
-	F_accelerate3d
+	FlagACPI Flag = 1 << iota
+	FlagIOAPIC
+	FlagRTCUseUTC
+	FlagCpuHotplug
```

Comment:
__[golint]__
* [golint] const FlagCpuHotplug should be FlagCPUHotplug

<!-- Comment by [SideCI](https://sideci.com) -->